src/components/NetworkSelector/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { motion } from 'framer-motion';
+import { useState } from 'preact/hooks';
+
+import { NetworkIcon } from '../NetworkIcon';
+import { NetworkIconType } from '../../hooks/useGetNetworkIcon';
+import { Networks, useNetwork } from '../../contexts/network';
+
+const capitalize = (str: string) => str.charAt(0).toUpperCase() + str.slice(1);
+
+export const NetworkSelector = () => {
```

Comment:
There is no click-away logic that would make the selector close when the user clicks elsewhere. Let's try to add that.